src/execute_request.jl

```diff
@@ -133,7 +134,7 @@ function helpcode(code::AbstractString)
         @compat if !haskey(Docs.keywords, Symbol(code_))
             return "Base.Docs.@repl $code_"
         elseif VERSION < v"0.5.0-dev+3831"
-            return "eval(:(Base.Docs.@repl \$(symbol(\"$code_\"))))"
+            return "eval(:(Base.Docs.@repl \$(Symbol(\"$code_\"))))"
```

Comment:
running in the jupyter notebook with @compat,
getting an error because that has not yet imported properly for eval(expr).
I have revert back to the original code. added a related test.